src/HotChocolate/Core/src/Execution/Processing/DefaultTransactionScopeHandler.cs

```diff
@@ -27,7 +27,8 @@ public virtual ITransactionScope Create(IRequestContext context)
                     new TransactionOptions
                     {
                         IsolationLevel = IsolationLevel.ReadCommitted
-                    }));
+                    },
+                    TransactionScopeAsyncFlowOption.Enabled));
```

Comment:
This is correct but you should make a separate PR for this. We can merge this change then into 11.3